src/app/collections/collections.tsx

```diff
@@ -45,8 +45,10 @@ export default class Collections extends React.Component<Props, State> {
     });
 
     const profileResponse = await getKiosks(this.props.account);
-    await fetchRatings(this.props.dimDestinyTrackerService, undefined, undefined, profileResponse, defs);
     this.setState({ profileResponse, defs });
+
+    await fetchRatings(defs, this.props.dimDestinyTrackerService, undefined, undefined, profileResponse);
+    this.forceUpdate();
```

Comment:
What I'm thinking of is more like this:

```typescript
const ratingsDb = await fetchRatings(...);
this.setState({ ratingsDb });
```

We shouldn't be calling `forceUpdate`.